src/main/java/org/wildfly/extension/elytron/SSLDefinitions.java

```diff
@@ -112,6 +116,12 @@
             .setFlags(AttributeAccess.Flag.RESTART_RESOURCE_SERVICES)
             .build();
 
+    static final SimpleAttributeDefinition ALIAS = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.ALIAS, ModelType.STRING, true)
```

Comment:
This alias attribute should be a filter compatible with the filtering KeyStore implementation we have within Elytron.